parm/postxconfig-NT-3drtma.txt

```diff
@@ -356,7 +356,7 @@ MAPS_PRMSL_ON_MEAN_SEA_LVL
 ?
 1
 tmpl4_0
-PRMSL
+MSLMA
 ?
```

Comment:
@EricJames-NOAA Please change line 360  "?" into "NCEP" for writing MSLMA in 3drtma products.